docs/metrics.md

```diff
@@ -50,7 +51,10 @@ Finally include that `ServiceMonitor` in your [Prometheus instance CRD](https://
 
 ### Stackdriver
 
-We don't yet support the [OpenCensus Stackdriver exporter](https://opencensus.io/exporters/supported-exporters/go/stackdriver/) but you can still use the Prometheus Stackdriver integration by following these [instructions](https://cloud.google.com/monitoring/kubernetes-engine/prometheus).
+We support the [OpenCensus Stackdriver exporter](https://opencensus.io/exporters/supported-exporters/go/stackdriver/). 
+In order to configure it you should enable Stackdriver Monitoring API in Google Cloud Console.
+
+Also you can use the Prometheus Stackdriver integration by following these [instructions](https://cloud.google.com/monitoring/kubernetes-engine/prometheus).
```

Comment:
Does it make sense to push prom+stackdriver at the same time we are saying use the stackdriver integration directly?

Is there any value add in using both layers together? if not, let's cut this piece I think.

Reply:
The only reason for that is to add golang metrics, like goroutines count, however it is not necessary to configure both at the same time.